models/catagory.js

```diff
@@ -0,0 +1,45 @@
+const { Model, DataTypes } = require('sequelize');
+const sequelize = require('../config/connection');
+
+
+class catagory extends Model{
+  checkUser(user_id) {
+    return user_id.compareSync(user_id,this.user);
+  }
```

Comment:
this wont do anything. user_id does not have a compareSync method for it and we wouldnt need to validate the user id as the catagory Model already has a foreign key '''user_id''' that links it to a specific user.